src/consts/parameter-consts.ts

```diff
@@ -1,6 +1,7 @@
 import { PlaygroundParameters } from "@/types";
 
 export const DEFAULT_ADDON_PARAMETERS: PlaygroundParameters = {
-  playgroundStoryId: "playground",
+  storyId: "playground",
   components: {},
+  editorTheme: null,
```

Comment:
Why null? Override the default?

Reply:
If it would have default light it would never query storybook's theme, enforcing the user to always declare it